httpclient/useragent.go

```diff
@@ -1,26 +0,0 @@
-package httpclient
-
-import (
-	"fmt"
-	"log"
-	"os"
-	"strings"
-
-	"github.com/hashicorp/terraform-plugin-sdk/v2/meta"
-)
-
-const uaEnvVar = "TF_APPEND_USER_AGENT"
-
-func TerraformUserAgent(version string) string {
```

Comment:
I do find this to be an appropriate helper for the SDK to offer so developers keep the UA's consistent, but an a package called `httpclient` with only this function does seem out of place for the SDK.